### ISSUE TITLE
Automatic Graph clustering?

### ISSUE THREAD

**Issue by pathikrit** [jacomyal/sigma.js#3]
http://stackoverflow.com/questions/9490003/web-based-large-scale-graph-drawing-library

**Comment by BransonGitomeh** [comment]
>The ForceAtlas2 example at sigmajs.org is displaying a

that link for `ForceAtlas2` is broken from my side, how far are you at this?... still coming soon?

**Comment by Yomguithereal** [comment]
If you are not afraid to try the v2 of the lib which is currently under development, you can also check [graphology](https://graphology.github.io/) which has Louvain community detection and which is used by the new version.

**Comment by paidge** [comment]
Thanks for your response and congratulation for your work ! I finally managed to implement [the algorithm](https://duniter.normandie-libre.fr/wotmap2/) to show the web of trust of the first crypto-currency co-created with a really universal basic income. Now I'm working on the layout but I noticed that all customisations are only possible with canvas renderer. I think I have to test the v2 or to wait for the next release.
